### PR TITLE
55.3.0

### PR DESCRIPTION
**Pull request by moiznalwalla**
https://enterprisedb.atlassian.net/browse/MIG-3962

### CODE REVIEW

File: product_docs/docs/migration_toolkit/55/06_building_toolkit.properties_file.mdx [hunk 1]
```diff
@@ -297,29 +297,53 @@ Migration Toolkit reads connection specifications for the source database from t
 -   The name of a privileged user
 -   The password associated with the specified user
 
-If you're connecting to a SQL Server database, `SRC_DB_URL` takes the form of a JTDS URL. For example:
+If you're connecting to a SQL Server database, `SRC_DB_URL` takes the form of a JTDS URL or Microsoft JDBC URL, as follows:
```
Comment (drothery-edb) [inline product_docs/docs/migration_toolkit/55/06_building_toolkit.properties_file.mdx:300]:
```suggestion
If you're connecting to a SQL Server database, `SRC_DB_URL` takes the form of a [JTDS URL](#jtds-url) or [Microsoft JDBC URL](microsoft-jdbc-url).
```

Comment (mlewandowski-edb) [reply product_docs/docs/migration_toolkit/55/06_building_toolkit.properties_file.mdx:300]: Is a # required before "microsoft-jdbc-url" in the hyperlink definition?

**Review by mlewandowski-edb — CHANGES_REQUESTED** [review]
Can we move the Defining a SQL Server URL  section to before the MySQL URL and Sybase section.  Also, I suggest reversing the order of the JTDS URL and Microsoft JDBC URL sections within the SQL Server URL section as it will be the preferred/recommended driver for SQL Server going forwarded.  The note in the Sybase URL about the jTDS driver would be relevant to the jTDS URL in SQL Server URL section too.

File: product_docs/docs/migration_toolkit/55/06_building_toolkit.properties_file.mdx (hunk 1 above)
Comment (moiznalwalla) [reply product_docs/docs/migration_toolkit/55/06_building_toolkit.properties_file.mdx:300]: Fixed.

**Comment by moiznalwalla** [comment]
> Can we move the Defining a SQL Server URL section to before the MySQL URL and Sybase section. Also, I suggest reversing the order of the JTDS URL and Microsoft JDBC URL sections within the SQL Server URL section as it will be the preferred/recommended driver for SQL Server going forwarded. The note in the Sybase URL about the jTDS driver would be relevant to the jTDS URL in SQL Server URL section too.

Incorporated.

**Review by mlewandowski-edb — APPROVED** [review]
I updated the tips related to the jTDS driver for SQL Server and Sybase ASE.

**Review by mlewandowski-edb — DISMISSED** [review]

**Review by mlewandowski-edb — COMMENTED** [review]
All of the installation pages seem to include this same first note in the Prerequisites section:

> Install EDB Postgres Advanced Server on the same host.

Is it really required that EDB Postgres Advanced Server be installed on the same host as Migration Toolkit?  I don't think it is for a source or target EPAS database.  Or is this note here to ensure that certain libraries are installed for use by Migration Toolkit?  If so, we may want to include some clarifying note.

**Comment by drothery-edb** [comment]
> All of the installation pages seem to include this same first note in the Prerequisites section:
> 
> > Install EDB Postgres Advanced Server on the same host.
> 
> Is it really required that EDB Postgres Advanced Server be installed on the same host as Migration Toolkit? I don't think it is for a source or target EPAS database. Or is this note here to ensure that certain libraries are installed for use by Migration Toolkit? If so, we may want to include some clarifying note.

@mlewandowski-edb We either have that as a prerequisite or this:

- Install EDB Postgres Advanced Server on a host that the product can connect to using a connection string. It doesn't need to be on the same host. See [Installing EDB Postgres Advanced Server](https://www.enterprisedb.com/docs/epas/latest/epas_inst_linux/).

We might have this all wrong. Would it make sense for MTK if we used the bullet that describes connecting using a connection string?

**Comment by mlewandowski-edb** [comment]
@drothery-edb 
> We either have that as a prerequisite or this:
> 
> * Install EDB Postgres Advanced Server on a host that the product can connect to using a connection string. It doesn't need to be on the same host. See [Installing EDB Postgres Advanced Server](https://www.enterprisedb.com/docs/epas/latest/epas_inst_linux/).
> 
> We might have this all wrong. Would it make sense for MTK if we used the bullet that describes connecting using a connection string?

Technically speaking, EPAS is not specifically required.  MTK could be used for migrations to PostgreSQL, albeit in a more limited way.   That being said, I like your revised version of the bullet.  I would go with that for now.

@zahid75 Do you have any opinion on this?

**Comment by drothery-edb** [comment]
> @drothery-edb
> 
> > We either have that as a prerequisite or this:
> > 
> > * Install EDB Postgres Advanced Server on a host that the product can connect to using a connection string. It doesn't need to be on the same host. See [Installing EDB Postgres Advanced Server](https://www.enterprisedb.com/docs/epas/latest/epas_inst_linux/).
> > 
> > We might have this all wrong. Would it make sense for MTK if we used the bullet that describes connecting using a connection string?
> 
> Technically speaking, EPAS is not specifically required. MTK could be used for migrations to PostgreSQL, albeit in a more limited way. That being said, I like your revised version of the bullet. I would go with that for now.
> 
> @zahid75 Do you have any opinion on this?

I changed the bullet in https://github.com/EnterpriseDB/docs/pull/3176 as we were making other change to the install topics in that PR. But, I wonder if the better solution is to just not include a prerequisite regarding epas.

**Pull request merged**